src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -222,20 +222,16 @@ export const AGGREGATIONS = {
   },
 } as const;
 
-assert(
-  AGGREGATIONS as Readonly<
-    {
-      [key in keyof typeof AGGREGATIONS]: {
-        parameters: Readonly<AggregateParameter[]>;
-        // null means to inherit from the column.
-        outputType: null | ColumnType;
-        isSortable: boolean;
-      };
-    }
-  >
-);
+assert(AGGREGATIONS as Readonly<{[key in keyof typeof AGGREGATIONS]: Aggregation}>);
```

Comment:
I was trying to avoid assert before because it effectively turns a whole bunch of potential errors.

Reply:
Looks like this one was already here right?

Reply:
Ah true. My bad.